### PR TITLE
add meetup 57

### PR DESCRIPTION
**Pull request by mzampetakis**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -87,6 +87,16 @@ controller('PastMeetupsController', function($scope) {
     return meetupDate > now;
   };
   $scope.MeetupList = [
+    {
+      Name: "Availability and monitoring / alerting",
```
Comment (kouts) [inline js/app.js:91]: To keep this consistent with https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/README.md ommit the spaces around the forward slash.

**Review by kouts — CHANGES_REQUESTED** [review]
Just a minor detail @mzampetakis , can you omit the spaces around the forward slash in "monitoring / alerting"?

File: js/app.js (hunk 1 above)
Comment (mzampetakis) [reply js/app.js:91]: Deliberately added the spaces around the slash because the text was displayed in a single line overflowing the UI's container box.

Comment (kouts) [reply js/app.js:91]: Got it! Thanks!

**Pull request merged**